src/js/files/components/__tests__/Manager.test.jsx

```diff
@@ -78,7 +78,7 @@ describe("<FileManager>", () => {
     });
 
     it("should change message if passed", async () => {
-        const account = createFakeAccount({ administrator_role: null });
+        const account = createFakeAccount({ administrator_role: "full" });
```

Comment:
Couple notes:

1. Use the `AdministratorRoles` enum. Enums are a useful way to limit string inputs to a set number of valid options and prevent typos from creating tough to debug errors
2. Make sure there are some tests where the user is not an administrator, but still has the upload permission. It is important to make sure that non-admin users that have upload permissions are still able to upload